docs/vcpkg-schema-definitions.schema.json

```diff
@@ -0,0 +1,715 @@
+      "type": "object",
+      "properties": {
+        "description": {
+          "$ref": "#/definitions/description-field"
+        },
+        "dependencies": {
+          "description": "Dependencies used by this feature.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/dependency"
+          }
+        },
+        "supports": {
+          "$ref": "#/definitions/platform-expression"
+        }
+      },
+      "patternProperties": {
+        "^\\$": {}
+      },
+      "required": [
+        "description"
+      ],
+      "additionalProperties": false
+    },
+    "filesystem-registry": {
+      "description": "A port registry backed by a git repo.",
+      "type": "object",
+      "properties": {
+        "kind": {
+          "type": "string",
+          "pattern": "filesystem"
+        },
+        "path": {
+          "type": "string",
+          "description": "The location on the filesystem where the registry is rooted."
+        }
+      },
+      "required": [
+        "kind",
+        "path"
+      ]
+    },
+    "info-block": {
+      "type": "object",
+      "properties": {
+        "version": {
+          "$ref": "#/definitions/semantic-version"
+        },
+        "id": {
+          "$ref": "#/definitions/slash-identifier"
+        },
+        "summary": {
+          "description": "A shorter description of the artifact.",
+          "type": "string"
+        },
+        "priority": {
+          "type": "integer",
+          "minimum": 0,
+          "default": 0
+        },
+        "description": {
+          "type": "string"
+        },
+        "options": {
+          "$ref": "#/definitions/string-array",
+          "default": []
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "version",
+        "id"
+      ]
+    },
+    "install": {
+      "if": {
+        "type": "object"
+      },
+      "then": {
+        "$ref": "#/definitions/install-instance"
+      },
+      "else": {
+        "type": "array",
+        "items": {
+          "$ref": "#/definitions/install-instance"
+        }
+      }
+    },
+    "install-instance": {
+      "type": "object",
+      "oneOf": [
+        {
+          "properties": {
+            "unzip": {
+              "$ref": "#/definitions/uri-or-uris"
+            },
+            "sha256": {
+              "$ref": "#/definitions/sha256"
+            },
+            "sha512": {
+              "$ref": "#/definitions/sha512"
+            },
+            "strip": {
+              "type": "integer",
+              "minimum": 0,
+              "default": 0
+            },
+            "transform": {
+              "type": "string",
+              "description": "An SED expression to run on the file names before extracting them."
+            },
+            "lang": {
+              "type": "string",
+              "description": "The locale under which this install should be applied."
```

Comment:
Note for discussion: why is this not a demand?

Reply:
Because they are intended to install 'on top' of each other while demands are supposed to be completely independent things.